sycl/include/CL/sycl/detail/pi.h

```diff
@@ -593,6 +593,16 @@ pi_result piPlatformGetInfo(pi_platform platform, pi_platform_info param_name,
 //
 // Device
 //
+///
+/// Create PI device from the given raw device handle (if the "device"
+/// points to null), or, vice versa, extract the raw device handle into
+/// the "handle" (if it was pointing to a null) from the given PI device.
+/// NOTE: The instance of the PI device created is retained.
+///
+pi_result piextDeviceInterop(
```

Comment:
To me the name of the API is not consistent with others. There is not action(verb) in the name.
Suggest renaming to:
piextDeviceConvert

Reply:
how about piextDeviceInteropConvert? 
@smaslov-intel Please give your opinion.

Reply:
Maybe piextDeviceRetainInterop? (see my other comment about "retain" semantics of this interface)

Reply:
piextDeviceInteropConvert sounds OK.

I don't think that piextDeviceRetainInterop matches what function does. Comment says that it creates of handle from another. Other "retain" functions take one input argument only while this on take one input and one output args.

Reply:
I am fine with "convert", but maybe drop "interop" then: piextDeviceConvert

Reply:
I feel piextDeviceConvert is not self-explanatory.
I will change it anyway, my understanding of PI/CL APIs is not that robust.